src/main/scala/cromwell/engine/WorkflowActor.scala

```diff
@@ -63,68 +65,75 @@ class WorkflowActor private(binding: WdlBinding, actualInputs: Map[FullyQualifie
   private def notStarted: Receive = {
     LoggingReceive {
 
-      case Start(backend) =>
+      case Start(system, backend) =>
         _primeMover = Option(sender())
```

Comment:
Sorry - the diff managed to not include the definition of _primeMover. W/ the WorkflowManager code, is _primeMover the WorkflowManager? (assuming we always go through a WM even in a 1 workflow run, I know that's an open Q)